src/main/kotlin/org/arctgkolbasy/bot/handler/HelpCommand.kt

```diff
@@ -1,35 +1,59 @@
 package org.arctgkolbasy.bot.handler
 
 import com.github.kotlintelegrambot.Bot
+import com.github.kotlintelegrambot.entities.InlineKeyboardMarkup
 import com.github.kotlintelegrambot.entities.Update
+import com.github.kotlintelegrambot.entities.keyboard.InlineKeyboardButton
 import org.arctgkolbasy.bot.user.User
 import org.arctgkolbasy.user.UserService
 import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.stereotype.Controller
+import java.util.*
 
 @Controller
 class HelpCommand(
-    val securedCommands: List<SecuredCommand>, userService: UserService,
+    val securedCommands: List<SecuredCommand>,
+    userService: UserService,
     @Qualifier("currentUserHolder")
     currentUserHolder: ThreadLocal<User?>,
 ) : SecuredCommand(
     userService = userService,
     currentUserHolder = currentUserHolder
 ) {
-    override fun getCommandName(): String = "help"
+    override fun getCommandName(): String = HELP_COMMAND
     override fun checkUserAccess(user: User) = true
+
     override fun handleUpdateInternal(user: User, bot: Bot, update: Update) {
         bot.sendMessage(
-            update.chatIdUnsafe(),
-            listOf(getCommandName())
-                .plus(securedCommands
-                    .filter { it.checkUserAccess(user) }
-                    .map { it.getCommandName() }
-                ).joinToString(
-                    prefix = "Available commands:\n",
-                    separator = "\n",
-                    transform = { "/$it" }
-                )
+            chatId = update.chatIdUnsafe(),
+            text = "Доступные команды:",
+            replyMarkup = getInlineKeyboardMarkup(user),
         )
     }
+
+    private fun getInlineKeyboardMarkup(user: User): InlineKeyboardMarkup {
+        val buttonsCatalogue: MutableList<InlineKeyboardButton> = mutableListOf()
+        val availableCommands = listOf(getCommandName())
+            .plus(securedCommands
+                .filter { it.checkUserAccess(user) }
+                .map {
+                    it.getCommandName()
+                })
+        availableCommands.forEach {
+            buttonsCatalogue.add(
+                InlineKeyboardButton.CallbackData(
+                    text = it
+                        .substring(0, 1).uppercase() + it.substring(1)
+                        .replace("_", " "),
+                    callbackData = it
+                        .replace("/", "")
+                )
+            )
+        }
+        return InlineKeyboardMarkup.create(buttonsCatalogue.chunked(3))
```

Comment:
здесь не нужно создавать и заполнять промежуточный buttonsCatalogue
1 то что ты делаешь в foreEach можно через map сделать
2 в котлине можно элегантнее мервую букву изменить
```suggestion
        val buttons = listOf(getCommandName())
            .plus(securedCommands.filter { it.checkUserAccess(user) })
            .map {
                InlineKeyboardButton.CallbackData(
                  text = it.getCommandName()
                      .replaceFirstChar { it.uppercase() }
                      .replace("_", " "),
                  callbackData = it.getCommandName()
                      .replace("/", "")
                )
            }
        return InlineKeyboardMarkup.create(buttons.chunked(3))
```